admin_guide/managing_projects.adoc

```diff
@@ -376,3 +379,53 @@ ifdef::openshift-enterprise[]
 # systemctl restart atomic-openshift-master-api atomic-openshift-master-controllers
 ----
 endif::[]
+
+ifdef::openshift-dedicated[]
+[[enforcing-different-linmits-on-users-projects]]
+== Enforcing Different Limits on Users' Projects
+
+If you are an {product-title} administrator and want to enforce different limits
+on the number of projects that your users can create, apply the appropriate
+label to the user.
+
+The admission controller must be configured for all possible values for the
+project limits. The default limit is `unlimited`.
+
+.Example Project Limit for Users
+
+----
+openshift.io/project-limit: '0'
+openshift.io/project-limit: '1'
+openshift.io/project-limit: '3'
+openshift.io/project-limit: '5'
+openshift.io/project-limit: '10'
+openshift.io/project-limit: '20'
+openshift.io/project-limit: '50'
+openshift.io/project-limit: '100'
+openshift.io/project-limit: 'unlimited'
+----
+
+[[managing-limits-and-quota-on-project-resources]]
+== Managing Limits and Quota on Project Resources
+
+There is a default project template available for any projects created by users
+on the cluster, which helps users from inadvertently consuming all of cluster
+and node resources. The project template is created and configured in the
+`dedicated-admin` namespace.
+
+There is an annotation on the project template that governs whether the template
```

Comment:
I'd rewrite this section to avoid "there is," to just show the parameter example once, and recenter it on "If you modify the default project template and want your changes to remain after you upgrade, ..."